### PR TITLE
Blog post for PyDelhi's dev sprint - 28/04/18

### PR DESCRIPTION
**Pull request by aaqaishtyaq**

### CODE REVIEW

File: content/2018-04-28-pydelhi-meetup.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Slug: pydelhi-meetup-28-april-2018 
+Date: 2018-04-28 
+Category: Meetup 
+Author: Aaqa Ishtyaq 
+Email: aaqaishtyaq@gmail.com
+Summary: ![Meetup at Knowlarity]({filename}/images/pydelhi-28-04-18-1.jpg) PyDelhi members coming together and meeting up to learn, collaborate, contribute.
+
+![Meetup at Knowlarity]({filename}/images/pydelhi-28-04-18-1.jpg)
+
+This time the meetup was at [Knowlarity](https://twitter.com/knowlarity), Ambience Mall, Gurugram. It began with everybody giving a brief introduction about their selves. And [Sanyam](https://twitter.com/ErSanyamKhurana) started the introductory session of what is a Dev sprint? During Dev sprint, Mentors/Contributors of Open source projects guides participants from generating development build to submit a patch for a bug. It is best opportunity to experience the culture of Open source contribution. People from newbie to experienced together contribute to project. During the session if you are stuck you can solve your confusions instantly by discussing with mentors. [Ambarish Gupta](https://twitter.com/ambarishKnow), CEO Knowlarity, also introduced himself and the company.
```
Comment (realslimshanky) [inline content/2018-04-28-pydelhi-meetup.md]: `During the session` -> `During the session,`

File: content/2018-04-28-pydelhi-meetup.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+The organisations who participated in this dev sprints along with their mentors were  :
+1. [Kivy](https://kivy.org) - [Akshay Arora](https://github.com/akshayaurora)
+2. [Coala](https://coala.io) - [Hemang Kumar](https://github.com/hemangsk) and [Rahul Jha](https://github.com/rj722)
+3. [Mozilla](https://mozilla.org) - [Sanyam Khurana](https://github.com/curiouslearner) for AddOn Server, Remo and Mozilla Central and [Shashank Kumar](https://github.com/realslimshanky) for AddOn Frontend
+4. [Python](https://python.org) - [Sanyam Khurana](https://github.com/curiouslearner) for CPython 
+ 
+
+After the introductory session, everyone chose their project in which they wanted to contribute and the dev sprint started. But there were a bunch of students who were unfamiliar with git and github and so [Priyank](https://twitter.com/priyankt68) took the initiative to teach git/github to those students.
```
Comment (realslimshanky) [inline content/2018-04-28-pydelhi-meetup.md]: `github and so` -> `GitHub, so`

File: content/2018-04-28-pydelhi-meetup.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ 
+
+After the introductory session, everyone chose their project in which they wanted to contribute and the dev sprint started. But there were a bunch of students who were unfamiliar with git and github and so [Priyank](https://twitter.com/priyankt68) took the initiative to teach git/github to those students. 
+The environment was full of zeal. Half way through the devsprint it was lunch time and Knowlarity had very generously sponsored lunch for all of us.
+
+![Devsprint at knowlarity]({filename}/images/pydelhi-28-04-18-2.jpg)
+
+Soon after lunch, everyone was back to their work. Before hiring and pitching session there was an inspirational talk by [Ajay Shrivastava](https://twitter.com/ajayshr), CTO Knowlarity, with a closing remark _"Work smart AND work hard. There is never a choice between the two. After all, it took Messi 17 years and 114 days to become an overnight success!"_. After his talk was hiring and pitching session. Anybody who’s looking for a job, want to hire someone, want a mentor, want to work on a project, etc. can just simply come on and let everybody know about it. If there’s somebody who’s interested in any participants pitch, can contact them directly.
```
Comment (realslimshanky) [inline content/2018-04-28-pydelhi-meetup.md]: `pitching session there` -> `pitching session, there`

File: content/2018-04-28-pydelhi-meetup.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+So that was all for the day. Devsprint concludes with 11 Issues, 5 Pull requests and lots of swag from Pydelhi and Github successfully. I hope you have enjoyed it and would join us at the [next meetup](https://www.meetup.com/pydelhi/).
```
Comment (realslimshanky) [inline content/2018-04-28-pydelhi-meetup.md]: `Devsprint concludes` -> 'Devsprint concluded`

**Review by realslimshanky — COMMENTED** [review]
The blog needs few minor improvements. Rest LGTM.

**Review by realslimshanky — APPROVED** [review]
LGTM

**Review by rajataaron — APPROVED** [review]
LGTM. Thanks @aaqaishtyaq for the blog.

**Comment by rajataaron** [comment]
@CuriousLearner can you merge these changes.

**Comment by vipulgupta2048** [comment]
@CuriousLearner Please merge the changes.

**Comment by CuriousLearner** [comment]
Not checking this locally and merging on the basis of review of @rajataaron

**Pull request merged**